firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -172,6 +172,11 @@ extension String {
                 tableName: "Bookmarks",
                 value: "Bookmarks",
                 comment: "Name of the top level bookmarks folder present in the folder selection menu of the`Edit Bookmark` screen")
+            public static let DeleteBookmark = MZLocalizedString(
+                key: "Bookmarks.Menu.DeleteBookmark.v132",
+                tableName: "Bookmarks",
+                value: "Delete Bookmark",
+                comment: "Title for the Delete Bookmark button in the Edit Bookmark screen.")
```

Comment:
```suggestion
                comment: "The title for the Delete Bookmark button, in the Edit Bookmark popup screen which is summoned from the main menu's Save submenu, which will delete the currently bookmarked site from the user's bookmarks.")
```
nit
I always go ham on comments so I avoid getting PRed on them. lolololol